python/cudf/cudf/tests/test_indexing.py

```diff
@@ -1299,13 +1299,32 @@ def test_loc_datetime_index(sli, is_dataframe):
         )
 
     gd_data = cudf.from_pandas(pd_data)
-
     expect = pd_data.loc[sli]
     got = gd_data.loc[sli]
-
     assert_eq(expect, got)
 
 
+@pytest.mark.parametrize(
+    "sli",
+    [
+        slice("2001", "2020"),
+        slice(None, "2020"),
```

Comment:
I didn't realize you could pass strings (or non-integer values) to `slice`... I guess this is handled by the `.loc[]`, but it feels funny to me.